moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1156,7 +1147,7 @@ as the new values that correspond to the group */
                               const GroupStateValidityCallbackFn& validCallback = GroupStateValidityCallbackFn(),
                               const kinematics::KinematicsQueryOptions& options = kinematics::KinematicsQueryOptions())
   {
-    return computeCartesianPath(group, traj, link, target, global_reference_frame, MaxEEFStep(max_step),
+    return computeCartesianPath(group, traj, link, target, global_reference_frame, MaxEEFStep(max_step, max_step),
```

Comment:
Why did you changed the semantics here from ignore to consider rotation? An explanation would be helpful...

Reply:
@mlautman Please confirm that this change in semantics was intended.

Reply:
@rhaschke I realize I never updated the documentation for this behavior change. I'm going to revert the change and maybe open a future pr with changes in default behavior